tests/cypress/support/commands.ts

```diff
@@ -247,7 +247,7 @@ Cypress.Commands.add('modifyDeployedApplication', (appName, clusterName='local')
 });
 
 // Create Role Template (User & Authentication)
-Cypress.Commands.add('createRoleTemplate', ({roleType='Global', roleName, newUserDefault='No', verbs, resources, apiGroups, nonResourcesURLs}) => {
+Cypress.Commands.add('createRoleTemplate', ({roleType='Global', roleName, newUserDefault='No', verbs, rules, apiGroups, nonResourcesURLs}) => {
```

Comment:
Verbs are part of rules, so not sure if it should be here

```suggestion
Cypress.Commands.add('createRoleTemplate', ({roleType='Global', roleName, newUserDefault='No', rules, apiGroups, nonResourcesURLs}) => {
```